qutebrowser/browser/downloadview.py

```diff
@@ -148,6 +148,7 @@ def _get_menu_actions(
         elif item.done:
             if item.successful:
                 actions.append(("Open", item.open_file))
+                actions.append(("Open directory", functools.partial(item.open_file, open_dir=True)))
```

Comment:
```suggestion
                actions.append(("Open directory", functools.partial(item.open_file, open_dir=True, cmdline=None)))
```

(see my comment for details)

Also please break the line so it's <79 characters.